### ISSUE TITLE
Question in train.py

### ISSUE THREAD

**Issue by verystrongjoe** [ikostrikov/pytorch-a3c#69]
https://github.com/ikostrikov/pytorch-a3c/blob/48d95844755e2c3e2c7e48bbd1a7141f7212b63f/train.py#L81

From this line including 3 following lines,

```
        R = torch.zeros(1, 1)
        if not done:
            value, _, _ = model((state.unsqueeze(0), (hx, cx)))
            R = value.detach()

```

Why does value tensor detach other than items in list of values?
